dataland-frontend/tests/component/testUtils/Keycloak.ts

```diff
@@ -2,6 +2,8 @@ import type Keycloak from "keycloak-js";
 import { type KeycloakInitOptions } from "keycloak-js";
 import { KEYCLOAK_ROLE_USER } from "@/utils/KeycloakUtils";
 
+export const KEYCLOAK_ROLE_PREMIUM_USER = "ROLE_PREMIUM_USER";
+
```

Comment:
Why should this be the only user for which this is defined here?

Reply:
legacy code - removed it